.github/workflows/ci.yml

```diff
@@ -1,13 +1,18 @@
-
 name: CI
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - main
+      - v6
```

Comment:
Do we need v6 and v7 in this repo?

Reply:
I think we need the release branch here (instead of 6 and 7)

Reply:
My idea was that we take this the same way as the core repository. So we're now on v6 (and CLI v6 supports Sequelize v5 and v6) and both go to v7 when dropping Node 10 support and CLI v7 will be focusing on supporting Sequelize v7 (but if we can maintain support for earlier versions of Sequelize that would be nice). Possibly a TypeScript migration for the CLI as well would be nice

Reply:
Makes sense!